rfcs/2021-05-23-broadcast.md

```diff
@@ -0,0 +1,50 @@
+# SMP agent broadcast
+
+## Problem
+
+Support agent message broadcast to multiple connections.
+
+It is done in ad-hoc way as part of the previous [groups proposal](./2021-05-15-groups.md) - this proposal defines broadcast as a separate agent primitive to simplify group management.
```

Comment:
link seems to be broken

Reply:
yes, it was renamed to a newer date, needs to be fixed